tensorflow/lite/micro/examples/hello_world/evaluate_test.cc

```diff
@@ -36,14 +50,17 @@ TfLiteStatus LoadFloatModelAndPerformInference() {
         model->version(), TFLITE_SCHEMA_VERSION);
   }
 
-  // This pulls in all the operation implementations we need
-  tflite::AllOpsResolver resolver;
+  HelloWorldOpResolver* op_resolver = nullptr;
+  if (tflite::GetOpResolver(&op_resolver) != kTfLiteOk) {
+    MicroPrintf("Could not resolve all the ops.");
+    return kTfLiteError;
+  }
 
   constexpr int kTensorArenaSize = 2056;
   uint8_t tensor_arena[kTensorArenaSize];
```

Comment:
add a comment that the arena size is simply a round number. The exact arena usage can be determined using the RecordingMicroInterpreter. And we should add that to the current .cc file in this or a follow-up PR.

And let's change the number to be 2000 or a suitably large number.